go/libkb/services.go

```diff
@@ -185,6 +185,18 @@ func (t *BaseServiceType) CanMakeNewProofs(mctx MetaContext) bool {
 	return !t.displayConf.CreationDisabled
 }
 
+func (t *BaseServiceType) CanMakeNewProofsSkipFeatureFlag(mctx MetaContext) bool {
```

Comment:
prefer `CanMakeNewProofs` just having a bool arg to toggle the behavior so we don't dupe the code